README.md

```diff
@@ -96,32 +96,37 @@ Snap deploys as a binary, which makes requirements quite simple. We've tested on
 
 ### Installation
 
-You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page. This isn't the comprehensive list of plugins, but they will help you get started. Right now, snap only supports Linux and OS X.
+You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page. This isn't the comprehensive list of plugins, but they will help you get started. Right now, snap only supports Linux and OS X (Darwin).
 
-If you're looking for the bleeding edge of snap, you can build it by cloning down the `master` branch. To build snap from source, you will need [Golang >= 1.4](https://golang.org) and [GNU Make](https://www.gnu.org/software/make/). More on building snap [here](./CONTRIBUTING.md).
+#### Building snap
+If you're looking for the bleeding edge of snap, you can build it by getting the `master` branch using `go get github.com/intelsdi-x/snap.git`. Otherwise you can just use the binaries. To build snap from source, you will need [Golang >= 1.4](https://golang.org) and [GNU Make](https://www.gnu.org/software/make/). For more on building snap check out [BUILD_AND_TEST.md](docs/BUILD_AND_TEST.md).
 
 ### Running snap
 
-Start a standalone snap agent:
+Start a standalone snap agent (snapd):
 
+If you cloned down the repository and aren't using just downloaded binaries, you can set the following:
 ```
-export SNAP_PATH=<snapdirectory>/build
-$SNAP_PATH/bin/snapd --plugin-trust 0 --log-level 1
+$ export SNAP_PATH=<snapDirectoryPath>/build
 ```
-This will bring up a snap agent without requiring plugin signing and set the logging level to debug. snap's REST API will be listening on port 8181.
+And then you can use `$SNAP_PATH` for your `<snapBinPath>`.
+```
+$ <snapBinPath>/bin/snapd --plugin-trust 0 --log-level 1
+```
+This will bring up a snap agent without requiring plugin signing and set the logging level to debug. snap's REST API will be listening on port 8181. To learn more about the snap agent and how to use it look at [SNAPD.md](docs/SNAPD.md) and/or run `<snapBinPath>/bin/snapd -h`.
 
 ### Running snap in tribe (cluster) mode
```

Comment:
I changed my mind here - it's good as is. No need to elaborate in this section given the much larger doc.